app/src/main/java/org/p2p/wallet/swap/jupiter/statemanager/SwapStateManager.kt

```diff
@@ -0,0 +1,24 @@
+package org.p2p.wallet.swap.jupiter.statemanager
+
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+
+class SwapStateManager(
+    private val handlers: Set<SwapStateHandler>,
+    private val scope: SwapCoroutineScope
```

Comment:
for what we create swapScope ? which difference between this and our IO scope ?

Reply:
scope is active while Swap feature is not closed
it is used to refresh Swap routes between different screens on Swap